src/media-segment-request.js

```diff
@@ -236,23 +229,6 @@ const handleSegmentResponse = ({
     segment.bytes = new Uint8Array(newBytes);
   }
 
-  // This is likely an FMP4 and has the init segment.
-  // Run through the CaptionParser in case there are captions.
-  if (captionParser && segment.map && segment.map.track && segment.map.track.video) {
-    // Initialize CaptionParser if it hasn't been yet
-    if (!captionParser.isInitialized()) {
```

Comment:
Everything I removed in this file was redundant and we never hit it. I think I messed up merging from master during the intial tba/lhls pr